files/image_config/monit/memory_checker

```diff
@@ -104,7 +131,13 @@ def main():
     parser.add_argument("threshold_value", type=int, help="threshold value in bytes")
     args = parser.parse_args()
 
-    check_memory_usage(args.container_name, args.threshold_value)
+    running_container_names = get_running_container_names()
+    if args.container_name in running_container_names:
```

Comment:
>running_container_names [](http://example.com/codeflow?start=30&length=23)

Add a sonic-mgmt test case?

Reply:
Working on the test case and will post the PR link at here.

Reply:
sonic-mgmt test case is posted: https://github.com/Azure/sonic-mgmt/pull/5823. If you are available, can you please help me review?